1.18/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends \
 
 # Install related tools
 ENV GOTESTSUM_V=1.8.1
-ENV GOCI_LINT_V=1.46.2
+ENV GOCI_LINT_V=1.45.2
```

Comment:
is this supposed to go backwards?

Reply:
Probably not. Let me check.

Reply:
I see what happened. The change went into the Dockerfile and not the template. I screwed up my review for this PR: https://github.com/CircleCI-Public/cimg-go/pull/183

I'll get this done in a different PR, and then fix this PR. Great catch!

Reply:
https://github.com/CircleCI-Public/cimg-go/pull/187